.circleci/config.yml

```diff
@@ -24,7 +18,6 @@ workflows:
               only: /^.*/
 env: &env
   working_directory: ~/project
-  resource_class: large
```

Comment:
I'm assuming this was for building the lib, and isn't needed for making the Docker image 🤞